app/components/resource-sidebar/template.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="sidebar-header">
+  {{#link-to 'stack' stack class="back-to-stack"}}
+    <i class="fa fa-chevron-left"></i>
+    <h3>{{resource.stack.handle}}</h3>
+  {{/link-to}}
+</div>
+
+<ul class="nav sidebar-nav">
+  <li class="title">
+    Apps
+    {{#aptible-ability scope="manage" user=user stack=stack as |hasAbility|}}
+      {{#if hasAbility}}
+        {{#link-to 'apps.new' stack }}
+          <i class="fa fa-plus"></i>
+        {{/link-to}}
+      {{/if}}
+    {{/aptible-ability}}
+  </li>
+
+  {{#each stack.apps as |app|}}
+    {{#link-to 'app' app tagName="li"}}
```

Comment:
I will leave this for now because it "works", but we'll need to update it to use `activating-item` once https://github.com/aptible/diesel.aptible.com/issues/196 is implemented.